lib/init.coffee

```diff
@@ -6,6 +6,14 @@ module.exports =
     rcFile:
       type: 'string'
       default: ''
+    showCodes:
+      description: "Show pylint codes? (E1101, etc.)"
```

Comment:
Consider instead something like:

title: "Show Pylint Codes"
description: "Displays the Pylint code in the Linter message  (e.g., E1101, etc.)"